draft-ietf-oauth-transaction-tokens.md

```diff
@@ -438,18 +439,12 @@ grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Atoken-exchange
 {: #figtxtokenrequest title="Example: Txn-Token Request"}
 
 ### Subject Token Types
-The `subject_token_type` parameter in the Txn-Token Request MAY have one of the following values:
+The `subject_token_type` parameter value MUST be a URI {{RFC3986}}. It MAY be one of the subject token types described in Section 3 of OAuth 2.0 Token Exchange {{RFC8693}}, or it MAY be set to the value:
 
-rfc9068
-: The `subject_token` is a JWT format OAuth 2.0 Access Token as defined in {{RFC9068}}
+`urn:ietf:params:txn-token:token-type:self-signed`
```

Comment:
I'm pretty sure you need an IANA section to register `urn:ietf:params:txn-token`.

Hint: https://www.iana.org/assignments/params/params.xhtml

Reply:
I have fixed this (changed to the OAuth namespace and added an IANA registry section.